.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Build
+
+on:
+  pull_request:
+    branches: ['**']
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        temurin_tag:
+          - 8u322-b06-jdk
+          - 11.0.14.1_1-jdk
+        architecture:
+          - linux/amd64
+          - linux/arm/v8
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker Hub
+        if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # https://github.com/docker/metadata-action
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            dwolla/jenkins-agent-core
+          # Docker tags based on the following events/attributes
+          tags: |
+            type=sha
```

Comment:
This needs to be tweaked so that it publishes tags along the Java version axis, e.g. `8-{sha}` or something like that.